### DISCUSSION TITLE
add option to tell us which ones resolve to internal ips

### DISCUSSION THREAD

**Discussion by 0xAwali** [projectdiscovery/dnsx#218] [General]
it's possible to add option to tell us which ones resolve to internal ips e.g. similar to  [hakfindinternaldomains](https://github.com/hakluke/hakfindinternaldomains)

```
cat subs.txt | dnsx -a -internal
sub.comapny.com [192.88.99.44] [Internal]
```

**Comment by brenocss** [comment]
https://github.com/projectdiscovery/dnsx/issues/139
`
echo sub.company.com | dnsx -json | jq 'select(.has_internal_ips==true)
`

**Comment by 0xAwali** [comment]
thanks @brenocss I didn't know that

**Comment by Mzack9999** [comment]
@0xAwali, this is already possible with `jq` filtering. I'm converting the issue into a discussion. Please let us know if you think we can make any further improvements. Thanks!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
